sass/_CategoryFacet.scss

```diff
@@ -1,6 +1,20 @@
 @import 'Variables';
 @import 'bourbon/bourbon';
 
+@mixin facetValueLabelPadding {
+  padding: 5px 19px 5px 20px;
```

Comment:
Does the right padding need to be `19px` or would `20px` work also? In which case, you do `padding: 5px 20px`.

Reply:
It's 19px to align the the value count with the normal facet value count.